week1/templates/aggregations.jinja2

```diff
@@ -31,7 +31,7 @@
         {% if agg.doc_count > 1 %}
           <div class="aggregation">
             <div><a class="text-blue-500 flex"
-                href="{{ url_for("search.query") }}?&query={{ query|e }}{{ applied_filters }}&filter.name=department.keyword&department.keyword.type=terms&department.keyword.key={{ agg.key }}">
+                href="{{ url_for("search.query") }}?&query={{ query|e }}{{ applied_filters }}&filter.name=department&department.type=terms&department.key={{ agg.key }}">
```

Comment:
you don't need to modify this if you set the query in this way:

```
"department": {
                "terms":{
                    "field": "department.keyword"
                }
            },
```

Or maybe I had to specify "department.keyword"  on the query because my mapping looks like: 
```
"department" : {
        "type" : "text",
        "fields" : {
          "keyword" : {
            "type" : "keyword",
            "ignore_above" : 256
          }
        }
      },
      ```

Reply:
I had to modified them as I had defined my `department` mapping as a `keyword` type rather than `text` 🙈